client/state/current-user/gravatar-status/selectors.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import has from 'lodash/has';
```

Comment:
Optional, but we now use `babel-plugin-lodash` that transforms this for us. So we can stick with:
```
import { has } from `lodash`;
```